### PR TITLE
Bug Fixes, One More Time Off Request Check, Time Off ID, and Remove Time Off Requests

### PR DESCRIPTION
**Pull request by yonimn2000**
See commit messages.

### CODE REVIEW

File: src/main/java/com/management/HumanResources/service/UpdateService.java
Comment (sterlingwalker) [inline src/main/java/com/management/HumanResources/service/UpdateService.java]: Lol so this might be a topic of discussion, this method is functionally identical to `denyTimeOff()` in that they both remove the time off from the employee. All time offs in the database don't have a `isReviewed` attribute. I'm sure we would write some temporary code that goes through each persons time off to add this. But then that might add some complexity to the client such as how would a reviewed time off be displayed? Then some logic has to be written to make sure a reviewed time off isn't denied again or approved 😆. I'll leave it up to you to decide if it's worth it to pursue the reviewed scenario.

**Review by sterlingwalker — COMMENTED** [review]
Really nice work! I think it's a much better idea to update the time off by `id`

File: src/main/java/com/management/HumanResources/service/UpdateService.java
Comment (yonimn2000) [reply src/main/java/com/management/HumanResources/service/UpdateService.java]: Yep, you are right. I had some issues when I started the sprint of getting validation errors upon approval/denial of requests but I later completely removed the validation as I thought it was unnecessary. I will remove this remove functionality.

Comment (sterlingwalker) [reply src/main/java/com/management/HumanResources/service/UpdateService.java]: Nice, and thank you!

**Review by sterlingwalker — APPROVED** [review]

**Pull request merged**
